tests/integration/test_authd/data/test_authd_valid_name_ip.yaml

```diff
@@ -212,3 +210,32 @@
       output:
         status: 'error'
         message: 'Invalid IP: 10.10.10.1'
+  -
+    name: 'Valid IPv6'
+    description: 'Try register an agent with valid IPv6: register'
+    test_case:
+    -
+      input: "OSSEC A:'user1' IP:'02db:4660:46af:e523:d05e:a62e:4ca7:8e58'"
+      output:
+        status: 'success'
+        name: 'user1'
+        ip: '02DB:4660:46AF:E523:D05E:A62E:4CA7:8E58'
+  -
+    name: 'Valid compressed IPv6'
+    description: 'Try register an agent with valid compressed IPv6: register'
+    test_case:
+    -
+      input: "OSSEC A:'user1' IP:'2001:db8:0:b::1A'"
+      output:
+        status: 'success'
+        name: 'user1'
+        ip: '2001:0DB8:0000:000B:0000:0000:0000:001A'
+  -
+    name: 'Invalid IPv6: 2 double colons'
```

Comment:
Maybe we could add more invalid test cases as in [enrollment test](https://github.com/wazuh/wazuh-qa/blob/2649ed3d6675abd314c6109d2fd08d9e9f9f64a3/tests/integration/test_enrollment/data/wazuh_enrollment_tests.yaml)

Reply:
Done in cec084b